### PR TITLE
Replace functions returning constants in common_functions

### PR DESCRIPTION
**Pull request by janssenhenning**
Replaced the functions `get_Ry2eV()` and similar functions for bohr to/from angström conversion with the corresponding constants in util.constants

This results in very slight differences for the Ry to eV conversion in the 7th decimal place for some kkr tests

### CODE REVIEW

**Review by PhilippRue — CHANGES_REQUESTED** [review]

File: masci_tools/io/parsers/kkrimp_parser_functions.py [hunk 1]
```diff
@@ -14,9 +14,10 @@
 Tools for the impurity caluclation plugin and its workflows
 """
 from numpy import array, ndarray, loadtxt
-from masci_tools.io.common_functions import search_string, open_general, get_version_info, get_Ry2eV, convert_to_pystd
+from masci_tools.io.common_functions import search_string, open_general, get_version_info, convert_to_pystd
 from masci_tools.io.parsers.kkrparser_functions import get_rms, find_warnings, get_charges_per_atom, get_core_states
 import traceback
+from masci_tools.util.constants import HTR_TO_EV
```
[the remark this answers is not in the document]
Comment (janssenhenning) [reply masci_tools/io/parsers/kkrimp_parser_functions.py]: No problem. Done with 7d5fcc0b150ee9f38aa289e9c7b0f9fd870a7879

File: masci_tools/io/parsers/kkrparser_functions.py [hunk 2]
```diff
@@ -840,7 +840,7 @@ def parse_kkr_outputfile(out_dict,
 
         try:
             result = get_Etot(outfile)
-            out_dict['energy'] = result[-1] * Ry2eV
+            out_dict['energy'] = result[-1] * HTR_TO_EV/2.0
```
[the remark this answers is not in the document]
Comment (janssenhenning) [reply masci_tools/io/parsers/kkrparser_functions.py]: Added (see comment above)

File: masci_tools/io/common_functions.py [hunk 3]
```diff
@@ -61,18 +61,6 @@ def get_alat_from_bravais(bravais, is3D=True):
...
-
-
-def get_aBohr2Ang():
-    return 1 / get_Ang2aBohr()
-
-
-def get_Ry2eV():
-    return 13.605693009
-
-
```
[the remark this answers is not in the document]
Comment (janssenhenning) [reply masci_tools/io/common_functions.py:75]: Readded the functions with DeprecationWarnings in a69e3fb92ba65dedf24738e4747234325241f288

**Review by PhilippRue — APPROVED** [review]
looks good to me 👍

**Pull request merged**